sul.yaml

```diff
@@ -1 +1,23 @@
-alerts: []
+---
+alerts:
+  - from: '2022-12-19T06:00:00-07:00'
+    to: '2023-01-03T22:00:00-07:00'
+    application_name: 'MyLibrary'
+    html: >
+      The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure
+      period (Wednesday, December 21, 2002 - Tuesday, January 3, 2023). More information about
```

Comment:
```suggestion
      period (Wednesday, December 21, 2022 - Tuesday, January 3, 2023). More information about
```

Reply:
I will confirm with @saseestone. This is the message she asked for.

Reply:
2002 seems like yesterday, but...